pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -408,7 +419,13 @@ buildStdenv.mkDerivation ({
   # tests were disabled in configureFlags
   doCheck = false;
 
+  # Generate build symbols once after the final build
+  # https://firefox-source-docs.mozilla.org/crash-reporting/uploading_symbol.html
   preInstall = ''
+    ./mach buildsymbols
+    mkdir -p $symbols/
+    cp mozobj/dist/*.crashreporter-symbols.zip $symbols/
```

Comment:
I don't particularly like putting it into `preInstall` but I want the build to create the symbols only after the second build phase in PGO builds.